docs/js/animated-trigger.js

```diff
@@ -16,6 +19,7 @@ const focusTrap = createFocusTrap('#animated-trigger', {
   onDeactivate: () => {
     container.classList.remove('is-active');
     trigger.classList.remove('is-triggered');
+    deactivatedFlag.classList.remove('is-hidden');
```

Comment:
Wondering why you added this here. It's already in onPostDeactivate, which is where we previously had the console message.

Reply:
Oh, I didn't see that...

Ok yeah sorry that isn't needed.